### PR TITLE
update azure armrest

### PR DESCRIPTION
**Pull request by d-m-u**
https://github.com/ManageIQ/manageiq-providers-azure/pull/395 needs core rest client updated and the cross repo specs on https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/124 are failing because smartstate also needs to upgrade

### CODE REVIEW

File: manageiq-smartstate.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "azure-armrest",      "~> 0.9.6"
+  spec.add_dependency "azure-armrest",      "~> 2.1.0"
```
Comment (agrare) [inline manageiq-smartstate.gemspec:23]: I think that's the rest-client version ;)
Can you update azure-armrest to just `0.10` (aka not 0.10.0)

Comment (d-m-u) [reply manageiq-smartstate.gemspec]: yeah sorry got it

**Comment by agrare** [comment]
Will merge once cross-repo tests are green

**Comment by agrare** [comment]
@d-m-u can you update this to match the azure repo?

**Pull request merged**